packages/synapse-react-client/src/lib/containers/auth/UsernamePasswordForm.tsx

```diff
@@ -58,16 +61,14 @@ export default function UsernamePasswordForm(props: UsernamePasswordFormProps) {
         type="submit"
         color="primary"
         variant="contained"
+        disabled={isLoading}
         sx={{
           height: '50px',
           mt: 4,
           mb: 2,
         }}
-        onClick={e => {
-          handleLogin(e)
-        }}
```

Comment:
Removing this handler has us fall back to the `form`'s `onSubmit`, which will let HTML 5 validation happen. The handler still has `preventDefault`, so the page won't reload.